test/parallel/test-require-symlink.js

```diff
@@ -1,53 +1,80 @@
 // Flags: --preserve-symlinks
 'use strict';
 const common = require('../common');
+
+const { execSync, spawn } = require('child_process');
+
+if (common.isWindows) {
+  // On Windows, creating symlinks requires admin privileges.
+  // We'll only try to run symlink test if we have enough privileges.
+  try {
+    const o = execSync('whoami /priv');
+    if (!o.includes('SeCreateSymbolicLinkPrivilege'))
+      common.skip('insufficient privileges');
+  } catch (e) {
+    common.skip('insufficient privileges');
+  }
+}
+
 const assert = require('assert');
-const path = require('path');
 const fs = require('fs');
-const { exec, spawn } = require('child_process');
+const path = require('path');
 const util = require('util');
+
+// Setup: Copy fixtures to tmp directory.
+
 const fixtures = require('../common/fixtures');
+const dirName = 'module-require-symlink';
+const fixtureSource = fixtures.path(dirName);
+const tmpDirTarget = path.join(common.tmpDir, dirName);
 
+// Copy fixtureSource to linkTarget recursively.
 common.refreshTmpDir();
 
-const linkTarget = fixtures.path('module-require-symlink',
-                                 'node_modules',
-                                 'dep2');
+function copyDir(source, target) {
+  fs.mkdirSync(target);
+  fs.readdirSync(source).forEach((entry) => {
+    const fullPathSource = path.join(source, entry);
+    const fullPathTarget = path.join(target, entry);
+    const stats = fs.statSync(fullPathSource);
+    if (stats.isDirectory()) {
+      copyDir(fullPathSource, fullPathTarget);
+    } else {
+      fs.writeFileSync(fullPathTarget, fs.readFileSync(fullPathSource));
```

Comment:
This can now be [`fs.copyFileSync()`](https://github.com/nodejs/node/pull/15034).